cards/src/main/resources/cards/custom/group0/other_death_knight/tgt/enchantment_midnight_ride.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Midnight Ride",
+  "type": "ENCHANTMENT",
+  "heroClass": "SPIRIT",
+  "rarity": "FREE",
+  "description": "Summon any minions you draw that were shuffled by Midnight Ride",
+  "trigger": {
+    "eventTrigger": {
+      "class": "CardDrawnTrigger",
+      "targetPlayer": "SELF",
+      "requiredAttribute": "RESERVED_BOOLEAN_2"
```

Comment:
Write a test and see if you need `RESERVED_BOOLEAN_2`. I know enchantments are removed on zone changes, but maybe not in the particular way that matters here.